api/src/modules/auth/authentication.service.ts

```diff
@@ -31,6 +40,8 @@ export class AuthenticationService {
     private readonly commandBus: CommandBus,
     private readonly eventBus: EventBus,
     private readonly passwordManager: PasswordManager,
+    @InjectRepository(BackOfficeSession)
```

Comment:
Minor detail:

We might inject the back office session repo only in the back office service, and move this service around. it might be more clean and would also allow us to encapsulate potential helper methods that we might need in the future, coupling back office related logic and data retrieval in a single point

Reply:
We can do it when we need to create the back office service and move forward now.